lunatrace/bsl/hasura/migrations/lunatrace/1649717845749_create-users-table-use-as-auth-table/up.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE public.users (
+    "id" uuid NOT NULL DEFAULT public.gen_random_uuid(),
+    "github_id" text UNIQUE,
+    "github_node_id" text UNIQUE,
+    "kratos_id" uuid,
+    PRIMARY KEY ("id"),
+    FOREIGN KEY ("kratos_id")
+       REFERENCES public.identities("id")
```

Comment:
you can set these up in line when you create the column with just `kratos_id uuid REFERENCES ....` but same difference!